client/src/state/viewMode/viewModeActions.js

```diff
@@ -0,0 +1,29 @@
+import ACTION_TYPES from './viewModeActionTypes';
+
+export function selectEditMode() {
+    return {
+        type: ACTION_TYPES.SELECT_EDIT,
+        payload: null,
```

Comment:
can just omit the payload for all these actions...
Also indent is all over the place for this file